### PR TITLE
Updating readme and changelog - sprint 20 to 23

### LINKED ISSUE #415: Update requirements documentation
Add OV screenshots? (discuss with Blaine)

### PR DESCRIPTION
**Pull request by adriane-cardozo**
- changelog: updated list of bug fixes and enhancements
- readme: 
 -- configuration ip: note about support for multiple OV instances
 -- configuration authlogindomain
 -- configuration ssl section: note about flask and cherrypy
 -- event service: link to the spec and some details about events being monitored
 -- composition service: new section added
 
Closes #415

### CODE REVIEW

**Comment by bsoutham** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -96,15 +96,20 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
...
-  Note: HPE OneView credentials are stored in clear-text. Make sure only authorized users can access this file.
 
   * **username**: HPE OneView's username
   
   * **password**: HPE OneView's password
 
+  * **authLoginDomain**: HPE OneView's authentication login domain. If not set, defaults to "Local".
+
+  Note: HPE OneView credentials are used only for authentication_mode set to "conf". They are stored in clear-text. Make sure only authorized users can access this file. When handling multiple OneView instances, make sure all instances have this username/password enabled.
+
```
[the remark this answers is not in the document]
Comment (adriane-cardozo) [reply README.md]: As discussed, I added a recommendation note for demo purposes only.

[the remark this answers is not in the document]
Comment (adriane-cardozo) [reply README.md]: Added " This configuration is recommended for demo purposes only."

File: README.md [hunk 2]
```diff
@@ -154,13 +162,28 @@ curl -X GET \
...
+Current implementation follows Redfish specification [DSP0266 version 1.5.0](https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.5.0.pdf). Event Service works only when authentication_mode is set to `conf`. As it connects directly to HPE OneView SCMB, the toolkit will request OneView to generate SCMB certs and download the certs to the correct location. The certs file are: **oneview\_ca**: OneView's CA cert file located at: `certs/oneview_ca.pem`. **scmb\_cert**: OneView's SCMB Client cert file located at: `certs/oneview_scmb.pem`. **scmb\_key**: OneView's SCMB Client key file located at: `certs/oneview_scmb.key`
+
+> In order to integrate properly with OneView, the OneView API 300 is required to be supported by OneView instance.
+
+Only alerts and events related to enclosures, racks and server hardware are being monitored and will generate the following Redfish events: ResourceAdded, ResourceUpdated and ResourceRemoved.
```
Comment (victorhugorodrigues) [inline README.md]: Maybe we could say that others resources could be added in the future to be monitored.

**Pull request merged**